src/ducks/balance/components/AccountRow.styl

```diff
@@ -96,3 +96,6 @@
     .AccountRow__logo
         opacity 0.5
 
+
+.AccountRow__ownersIcon
+    margin-right 0.25rem
```

Comment:
Maybe spacing_values could be used here ?

Reply:
This value does not exist in `spacing_values`. `xs` is `0.5rem`. But since we have `xxl`, maybe we can have `xxs`?